spec/decorators/comment_decorator_spec.rb

```diff
@@ -46,4 +46,12 @@
       expect(comment.published_timestamp).to eq(comment.created_at.utc.iso8601)
     end
   end
+
+  describe "#published_at_int" do
+    let(:comment) { build(:comment) }
+
+    it "returns the creation date as an integer" do
+      expect(comment.decorate.published_at_int).to eq(comment.created_at.to_i)
```

Comment:
this works because `nil.to_i` is 0 for both. As we `build()` instead of `create()` the comment will not have a `created_at` populated

Reply:
Great point, added create and a nil check